src/lib/proof-system/prover-keys.ts

```diff
@@ -53,7 +53,7 @@ function parseHeader(
   methods: MethodInterface[],
   header: SnarkKeyHeader
 ): CacheHeader {
-  let hash = Pickles.util.fromMlString(header[1][2][7]);
+  let hash = Pickles.util.fromMlString(header[1][2][6]);
```

Comment:
so, did the ocaml type change? if yes it would be nice to also change `MlSnarkKeysHeader`

Reply:
yep https://github.com/o1-labs/o1js/pull/1804/commits/8ab009cfd35154cb10bfca2cb34cef6a3f76f05c
https://github.com/MinaProtocol/mina/commit/93d16e26a437a9366fa40d8d0f422fc00a8a6063#diff-e97cc66a36b931a3bf0e75bcead151b392188d176a3c71e7408756487b8ff5d1L133